release/LICENSE.txt

```diff
@@ -0,0 +1 @@
+This work is licenced under CC BY 4.0, except where otherwise stated. The City of Helsinki logo is a registered trademark. The Helsinki Grotesk Typeface is a proprietary typeface licensed by Camelot Typefaces.
```

Comment:
This license seem to conflict a bit with the root license. Why it is Creative Commons license?

Reply:
I just copied the one from the Icon kit 😄 Also the typeface license mention does not apply to icons.